src/pages/Promotion/Content.js

```diff
@@ -25,111 +28,189 @@ export default class Content extends React.Component {
+                        <Col xs={24} sm={24} md={10} lg={8} xl={7} xxl={6}>
+                            <Card className="part5-section">
+                                <div className="part5-section-title">
+                                    <img src="/assets/icon/promotion/premium.svg"/><span>Multiple Premium Plans</span>
                                 </div>
-                                <div className="part5-section-title">Multiple Premium Plans</div>
                                 <div className="part5-section-subtitle">will satisfy all your distinctive needs with selected features and access.</div>
                                 <div className="part5-section-link">
                                     <span>More Information</span>
                                     <a><img src="/assets/icon/promotion/arrow.svg"/></a>
                                 </div>
-                            </div>
+                            </Card>
                         </Col>
-                        <Col span="12" className="part5-even">
-                            <div className="part5-section">
-                                <div className="part5-section-img">
-                                    <img src="/assets/icon/promotion/edit.svg"/>
+                        <Col xs={24} sm={24} md={10} lg={8} xl={7} xxl={6}>
+                            <Card className="part5-section">
+                                <div className="part5-section-title">
+                                    <img src="/assets/icon/promotion/edit.svg"/><span>Paid by features</span>
                                 </div>
-                                <div className="part5-section-title">Paid by features</div>
                                 <div className="part5-section-subtitle">make sure you make the most of it by building your unique plan.</div>
                                 <div className="part5-section-link">
                                     <span>Check It Out</span>
                                     <a><img src="/assets/icon/promotion/arrow.svg"/></a>
                                 </div>
-                            </div>
+                            </Card>
                         </Col>
+                        <Col xs={0} sm={0} md={2} lg={4} xl={5} xxl={6}></Col>
                     </Row>
                 </Row>
                 <Row className="part6">
-
+                    <div className="part6-header">- Future -</div>
+                    <div className="part6-title">Unlock New Features Overtime</div>
+                    <div className="part6-subtitle">Your continous-use of Dr.Chain can also unlock new features.</div>
+                    <div className="part6-background">
+                        <Col xs={0} sm={0} md={2} lg={4} xl={5} xxl={5}></Col>
+                        <Col xs={24} sm={24} md={17} lg={15} xl={14} xxl={14}>
+                            <img src="/assets/images/promotion_bot.png" alt="heart"/>
```

Comment:
seems this png is not responsive with the zoom in/out of the browser, is there any possibility we could explore to fix this?